### DISCUSSION TITLE
制定英文版UIGF标准原文

### DISCUSSION THREAD

**Discussion by TremblingMoeNew** [UIGF-org/.github#2] [General]
UIGF标准即将对多语言环境提供支持。
出于国际化与扩展支持工具的需求以及在此过程中的交流需要，应当确定UIGF标准的英文版本原文。

**Comment by TremblingMoeNew** [comment]
我之前翻译过一版，[在我自己的网站上有](https://dodocotales.cc/#/en-us/wish-log-formats/universal-format/uigf-standard)。
不过建议先对中文版标准做一下修订，现在这个看着还是有一点点乱
@Lightczx @biuuu @sunfkny @Scighost

**Reply by Lightczx** [reply]
https://github.com/UIGF-org/UIGF-org.github.io/commit/7ddacde4d8ec719e45027af2ddf030118a4973c1

@TremblingMoeNew  FYI
